app/Http/Controllers/v1/ListController.php

```diff
@@ -43,23 +64,27 @@ protected function getItems(Request $request, string $listId): Collection
             // the URL.
             $query->where(function ($query) use ($itemIds) {
                 foreach ($itemIds as $itemId) {
-                    $item = ListItem::createFromUrlParameter($itemId);
+                    $item = ListItem::createFromString($itemId);
                     $this->idQuery($query, $item, true);
                 }
             });
         }
 
         $items = $query->orderBy('changed_at', 'DESC')
-            ->select($this->idColumn)
-            ->pluck($this->idColumn);
+            ->get(['material', 'collection']);
```

Comment:
This is getting messy. I wonder if we should have used a model class instead?

Probably too late, but `ListItem::getByMaterialId()`, etc. to hide the back and forth conversion would move concerns about the database schema out of the controllers. Donno if we could just make ListItem a model class.